Grading at 12/08/2023:15:44:30.105710
For commit ID c939f33c88c86b2210078cd26b9a9afd7a4092cc
Grading at Fri Dec  8 10:41:29 AM EST 2023
Doing some basic checking for your submission

Step 1
Checking that cyoa-step1 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4 cyoa-step1.o cyoa-step2.o cyoa-step3.o cyoa-step4.o cyoa.o
g++    -c -o cyoa-step1.o cyoa-step1.cpp
g++ -std=gnu++11 -pedantic -Wall -Werror -ggdb3 -c cyoa.cpp
g++ -o cyoa-step1 cyoa-step1.o cyoa.o
rm cyoa.o cyoa-step1.o
The code compiled (good)
Executable cyoa-step1 exists (good)
------------------------------------
Now running YOUR tests for step 1
------------------------------------


1.1: Current test is ./cyoa-step1 story1 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.2: Current test is ./cyoa-step1 test/step3_test7 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.3: Current test is ./cyoa-step1 story2 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.4: Current test is ./cyoa-step1 brokenstory < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open story.txt file

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.5: Current test is ./cyoa-step1 story1 story2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid input command

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.6: Current test is ./cyoa-step1  < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid input command

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.7: Current test is ./cyoa-step1 test/NoDelimiter < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.8: Current test is ./cyoa-step1 test/no_story < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open story.txt file

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.9: Current test is ./cyoa-step1 test/pageDeclareOrder < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.10: Current test is ./cyoa-step1 test/size_t_2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.11: Current test is ./cyoa-step1 test/typeOfPage < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story page input format invalid
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.12: Current test is ./cyoa-step1 test/WLnoChoice < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Logic error when adding choice not to type 'N'

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.13: Current test is ./cyoa-step1 test/number < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.14: Current test is ./cyoa-step1 test/pageNum < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.15: Current test is ./cyoa-step1 test/story1 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open story.txt file

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.16: Current test is ./cyoa-step1 test/filename < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open File
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.17: Current test is ./cyoa-step1 test/orderNum < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Page is not yet declared

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.18: Current test is ./cyoa-step1 test/size_t < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.19: Current test is ./cyoa-step1 story3 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.20: Current test is ./cyoa-step1 test/step4_test1 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.21: Current test is ./cyoa-step1 test/step4_test2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.22: Current test is ./cyoa-step1 test/step4_test3 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.23: Current test is ./cyoa-step1 test/step4_test4 < /dev/null
-----------------------------
We considered this a failure case.
But you did not (exit status was EXIT_SUCCESS).


1.24: Current test is ./cyoa-step1 test/step4_test5 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.25: Current test is ./cyoa-step1 test/step3_test1 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.26: Current test is ./cyoa-step1 test/step3_test2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.27: Current test is ./cyoa-step1 test/step3_test3 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.28: Current test is ./cyoa-step1 test/step3_test4 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.29: Current test is ./cyoa-step1 test/step2_test1 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.30: Current test is ./cyoa-step1 test/step2_test2 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.31: Current test is ./cyoa-step1 test/step2_test3 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.32: Current test is ./cyoa-step1 test/step2_test4 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.33: Current test is ./cyoa-step1 test/step2_test5 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.34: Current test is ./cyoa-step1 test/step3_test5 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


1.35: Current test is ./cyoa-step1 test/step3_test7 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean
No request to check for intermediate deadline completion on step 1

Step 2
Checking that cyoa-step2 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4 cyoa-step1.o cyoa-step2.o cyoa-step3.o cyoa-step4.o cyoa.o
g++    -c -o cyoa-step2.o cyoa-step2.cpp
g++ -std=gnu++11 -pedantic -Wall -Werror -ggdb3 -c cyoa.cpp
g++ -o cyoa-step2 cyoa-step2.o cyoa.o
rm cyoa.o cyoa-step2.o
The code compiled (good)
Executable cyoa-step2 exists (good)
------------------------------------
Now running YOUR tests for step 2
------------------------------------


2.1: Current test is ./cyoa-step2  < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid input command

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.2: Current test is ./cyoa-step2 story1 story2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid input command

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.3: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input0.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.4: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.5: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input2.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.6: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input3.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.7: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input3.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.8: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input5.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.9: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input6.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.10: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input7.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.11: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input8.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.12: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input9.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.13: Current test is ./cyoa-step2 story1  <  /usr/local/ece551/cyoa/story1/inputs/input10.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.14: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input0.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.15: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.16: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input2.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.17: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input3.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.18: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input3.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.19: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.20: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input6.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.21: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input7.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.22: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input8.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.23: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input9.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.24: Current test is ./cyoa-step2 test/step3_test1  <  /usr/local/ece551/cyoa/story1/inputs/input10.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.25: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input0.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.26: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.27: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input3.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.28: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input4.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.29: Current test is ./cyoa-step2 story2  <  story2/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.30: Current test is ./cyoa-step2 story2  <  story2/inputs/input6.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Input is not good

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.31: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input7.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.32: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input8.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.33: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input9.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.34: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input10.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.35: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input11.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.36: Current test is ./cyoa-step2 story2  <  /usr/local/ece551/cyoa/story1/inputs/input12.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.37: Current test is ./cyoa-step2 story1  <  nothing
-----------------------------
You considered this a failure case:
/dev/fd/3: line 528: nothing: No such file or directory
But we did not.


2.38: Current test is ./cyoa-step2 story3  <  /usr/local/ece551/cyoa/story3/inputs/input15.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.39: Current test is ./cyoa-step2 story3  <  /usr/local/ece551/cyoa/story3/inputs/input16.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.40: Current test is ./cyoa-step2 story3  <  /usr/local/ece551/cyoa/story3/inputs/input17.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.41: Current test is ./cyoa-step2 story3  <  /usr/local/ece551/cyoa/story3/inputs/input18.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.42: Current test is ./cyoa-step2 story3  <  /usr/local/ece551/cyoa/story3/inputs/input19.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.43: Current test is ./cyoa-step2 story3  <  /usr/local/ece551/cyoa/story2/inputs/input0.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Input is not good

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.44: Current test is ./cyoa-step2 story3  <  /usr/local/ece551/cyoa/story1/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.45: Current test is ./cyoa-step2 test/step4_test3  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.46: Current test is ./cyoa-step2 test/step4_test3  <  /usr/local/ece551/cyoa/story3/inputs/input2.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.47: Current test is ./cyoa-step2 test/step4_test4  <  /usr/local/ece551/cyoa/story3/inputs/input3.txt
-----------------------------
We considered this a failure case.
But you did not (exit status was EXIT_SUCCESS).


2.48: Current test is ./cyoa-step2 test/step4_test3  <  /usr/local/ece551/cyoa/story3/inputs/input4.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.49: Current test is ./cyoa-step2 test/step4_test5  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.50: Current test is ./cyoa-step2 test/step4_test3  <  /usr/local/ece551/cyoa/story3/inputs/input6.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.51: Current test is ./cyoa-step2 test/step4_test5  <  /usr/local/ece551/cyoa/story3/inputs/input7.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.52: Current test is ./cyoa-step2 test/step4_test4  <  /usr/local/ece551/cyoa/story3/inputs/input8.txt
-----------------------------
We considered this a failure case.
But you did not (exit status was EXIT_SUCCESS).


2.53: Current test is ./cyoa-step2 test/step4_test4  <  /usr/local/ece551/cyoa/story3/inputs/input15.txt
-----------------------------
We considered this a failure case.
But you did not (exit status was EXIT_SUCCESS).


2.54: Current test is ./cyoa-step2 test/step4_test1  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.55: Current test is ./cyoa-step2 test/step4_test1  <  /usr/local/ece551/cyoa/story3/inputs/input17.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.56: Current test is ./cyoa-step2 test/step3_test6  <  /usr/local/ece551/cyoa/story3/inputs/input18.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.57: Current test is ./cyoa-step2 test/step3_test6  <  /usr/local/ece551/cyoa/story3/inputs/input19.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.58: Current test is ./cyoa-step2 test/step3_test7  <  /usr/local/ece551/cyoa/story3/inputs/input0.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.59: Current test is ./cyoa-step2 test/step3_test7  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.60: Current test is ./cyoa-step2 test/step4_test1  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.61: Current test is ./cyoa-step2 test/step4_test1  <  /usr/local/ece551/cyoa/story3/inputs/input2.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.62: Current test is ./cyoa-step2 test/step4_test1  <  /usr/local/ece551/cyoa/story3/inputs/input3.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.63: Current test is ./cyoa-step2 test/step4_test1  <  /usr/local/ece551/cyoa/story3/inputs/input4.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.64: Current test is ./cyoa-step2 test/step4_test1  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.65: Current test is ./cyoa-step2 test/step3_test4  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.66: Current test is ./cyoa-step2 test/step2_test1  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
The story does not have a Lose page

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.67: Current test is ./cyoa-step2 test/step2_test2  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.68: Current test is ./cyoa-step2 test/step2_test3  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


2.69: Current test is ./cyoa-step2 test/step2_test4  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Page is missing or is not referenced

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.70: Current test is ./cyoa-step2 test/step2_test5  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Page is missing or is not referenced

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean
No request to check for intermediate deadline completion on step 2

Step 3
Checking that cyoa-step3 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4 cyoa-step1.o cyoa-step2.o cyoa-step3.o cyoa-step4.o cyoa.o
g++    -c -o cyoa-step3.o cyoa-step3.cpp
g++ -std=gnu++11 -pedantic -Wall -Werror -ggdb3 -c cyoa.cpp
g++ -o cyoa-step3 cyoa-step3.o cyoa.o
rm cyoa.o cyoa-step3.o
The code compiled (good)
Executable cyoa-step3 exists (good)
------------------------------------
Now running YOUR tests for step 3
------------------------------------


3.1: Current test is ./cyoa-step3 story1 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.2: Current test is ./cyoa-step3 story2 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.3: Current test is ./cyoa-step3 story1 story2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid input command

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.4: Current test is ./cyoa-step3  < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid input command

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.5: Current test is ./cyoa-step3 test/NoDelimiter < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.6: Current test is ./cyoa-step3 test/no_story < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open story.txt file

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.7: Current test is ./cyoa-step3 test/pageDeclareOrder < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.8: Current test is ./cyoa-step3 test/size_t_2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.9: Current test is ./cyoa-step3 test/typeOfPage < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story page input format invalid
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.10: Current test is ./cyoa-step3 test/WLnoChoice < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Logic error when adding choice not to type 'N'

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.11: Current test is ./cyoa-step3 test/number < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.12: Current test is ./cyoa-step3 test/pageNum < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.13: Current test is ./cyoa-step3 test/story1 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open story.txt file

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.14: Current test is ./cyoa-step3 test/filename < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open File
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.15: Current test is ./cyoa-step3 test/orderNum < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Page is not yet declared

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.16: Current test is ./cyoa-step3 test/size_t < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.17: Current test is ./cyoa-step3 story3 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.18: Current test is ./cyoa-step3 test/step4_test1 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.19: Current test is ./cyoa-step3 test/step4_test2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.20: Current test is ./cyoa-step3 test/step4_test3 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.21: Current test is ./cyoa-step3 test/step4_test4 < /dev/null
-----------------------------
We considered this a failure case.
But you did not (exit status was EXIT_SUCCESS).


3.22: Current test is ./cyoa-step3 test/step4_test5 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.23: Current test is ./cyoa-step3 test/step3_test1 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.24: Current test is ./cyoa-step3 test/step3_test2 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.25: Current test is ./cyoa-step3 test/step3_test3 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.26: Current test is ./cyoa-step3 test/step3_test4 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.27: Current test is ./cyoa-step3 test/step2_test1 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
The story does not have a Lose page

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.28: Current test is ./cyoa-step3 test/step2_test2 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.29: Current test is ./cyoa-step3 test/step2_test3 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.30: Current test is ./cyoa-step3 test/step2_test4 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Page is missing or is not referenced

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.31: Current test is ./cyoa-step3 test/step3_test6 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.32: Current test is ./cyoa-step3 test/step3_test7 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


3.33: Current test is ./cyoa-step3 test/step3_test8 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean
No request to check for intermediate deadline completion on step 3

Step 4
Checking that cyoa-step4 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4 cyoa-step1.o cyoa-step2.o cyoa-step3.o cyoa-step4.o cyoa.o
g++    -c -o cyoa-step4.o cyoa-step4.cpp
g++ -std=gnu++11 -pedantic -Wall -Werror -ggdb3 -c cyoa.cpp
g++ -o cyoa-step4 cyoa-step4.o cyoa.o
rm cyoa-step4.o cyoa.o
The code compiled (good)
Executable cyoa-step4 exists (good)
------------------------------------
Now running YOUR tests for step 4
------------------------------------


4.1: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story3/inputs/input0.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.2: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.3: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story3/inputs/input2.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.4: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story3/inputs/input3.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.5: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story3/inputs/input4.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.6: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story3/inputs/input19.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.7: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story2/inputs/input0.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Input is not good

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.8: Current test is ./cyoa-step4 story3  <  /usr/local/ece551/cyoa/story1/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.9: Current test is ./cyoa-step4 test/step4_test3  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.10: Current test is ./cyoa-step4 test/step4_test3  <  /usr/local/ece551/cyoa/story3/inputs/input2.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Story.txt format for variable incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.11: Current test is ./cyoa-step4 test/step4_test5  <  /usr/local/ece551/cyoa/story3/inputs/input18.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.12: Current test is ./cyoa-step4 test/step4_test5  <  /usr/local/ece551/cyoa/story3/inputs/input19.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.13: Current test is ./cyoa-step4 test/step4_test5  <  /usr/local/ece551/cyoa/story3/inputs/input0.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.14: Current test is ./cyoa-step4 test/step4_test5  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.15: Current test is ./cyoa-step4 test/step3_test1  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.16: Current test is ./cyoa-step4 test/step3_test1  <  /usr/local/ece551/cyoa/story3/inputs/input2.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.17: Current test is ./cyoa-step4 test/step3_test1  <  /usr/local/ece551/cyoa/story3/inputs/input3.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.18: Current test is ./cyoa-step4 test/step4_test5  <  /usr/local/ece551/cyoa/story3/inputs/input4.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Out of range error

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.19: Current test is ./cyoa-step4 test/step3_test5  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.20: Current test is ./cyoa-step4 test/step3_test5  <  /usr/local/ece551/cyoa/story3/inputs/input6.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.21: Current test is ./cyoa-step4 test/step3_test5  <  /usr/local/ece551/cyoa/story3/inputs/input7.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.22: Current test is ./cyoa-step4 test/step3_test1  <  /usr/local/ece551/cyoa/story3/inputs/input8.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.23: Current test is ./cyoa-step4 test/step3_test2  <  /usr/local/ece551/cyoa/story3/inputs/input15.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.24: Current test is ./cyoa-step4 test/step3_test2  <  /usr/local/ece551/cyoa/story3/inputs/input16.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Declaration order incorrect

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.25: Current test is ./cyoa-step4 test/step4_test2  <  /usr/local/ece551/cyoa/story3/inputs/input17.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.26: Current test is ./cyoa-step4 test/step3_test5  <  /usr/local/ece551/cyoa/story3/inputs/input18.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.27: Current test is ./cyoa-step4 test/step4_test2  <  /usr/local/ece551/cyoa/story3/inputs/input19.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.28: Current test is ./cyoa-step4 test/step4_test2  <  /usr/local/ece551/cyoa/story2/inputs/input0.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.29: Current test is ./cyoa-step4 test/step4_test2  <  /usr/local/ece551/cyoa/story1/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid digits from story.txt for page 

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.30: Current test is ./cyoa-step4 test/step4_test0  <  /usr/local/ece551/cyoa/story3/inputs/input4.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Fail to open story.txt file

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.31: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.32: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input6.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.33: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input7.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.34: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input8.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.35: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input15.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.36: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input17.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.37: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input18.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.38: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input19.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Input is not good

-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.39: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input0.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.40: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.41: Current test is ./cyoa-step4 test/step3_test6  <  /usr/local/ece551/cyoa/story3/inputs/input1.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.42: Current test is ./cyoa-step4 test/step3_test6  <  /usr/local/ece551/cyoa/story3/inputs/input2.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.43: Current test is ./cyoa-step4 test/step3_test6  <  /usr/local/ece551/cyoa/story3/inputs/input3.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.44: Current test is ./cyoa-step4 test/step4_test6  <  /usr/local/ece551/cyoa/story3/inputs/input4.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.45: Current test is ./cyoa-step4 test/step3_test6  <  /usr/local/ece551/cyoa/story3/inputs/input5.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.46: Current test is ./cyoa-step4 test/step3_test7  <  /usr/local/ece551/cyoa/story3/inputs/input6.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.47: Current test is ./cyoa-step4 test/step3_test7  <  /usr/local/ece551/cyoa/story3/inputs/input7.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean


4.48: Current test is ./cyoa-step4 test/step3_test7  <  /usr/local/ece551/cyoa/story3/inputs/input8.txt
-----------------------------
Your output looks good. Lets check valgrind
Valgrind was clean
Start time 1702050089 seconds after epoch
Current time 1702050269 seconds after epoch
You have used more than 2 minutes of compute time. Exiting.

Overall Grade: NO GRADE YET
